### PR TITLE
Remfiles/#54/55

### PR DESCRIPTION
**Pull request by Allislove**
### Description

Removed unused files - 

fixes 
issues #54 #54

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [X] The branch is updated with the last dev branch changes.

### CODE REVIEW

**Comment by Allislove** [comment]
This issue. fixed #55 and #54

File: package.json [hunk 1]
```diff
@@ -4,9 +4,11 @@
   "private": true,
   "dependencies": {
     "axios": "0.24.0",
+    "bootstrap": "^5.1.3",
     "notiflix": "3.2.2",
     "query-string": "7.0.1",
     "react": "17.0.2",
+    "react-bootstrap": "^2.2.0",
```
Comment (heliomar-pena) [inline package.json:11]: Why have you installed react-bootstrap & bootstrap?

**Review by heliomar-pena — APPROVED** [review]

**Review by e-azocar — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Allislove) [reply package.json:11]: The dev branch was with problems, when I fetch this!! The app was with error cuz the react pagínate! So I just installed react pagínate and this update the bootstrap too.

Comment (heliomar-pena) [reply package.json:11]: I was checked and React Paginate package.json doesn't include bootstrap https://github.com/AdeleD/react-paginate/blob/master/package.json , what if remove bootstrap from this pull request and you create an issue with a screenshot of the error that you're getting?

**Comment by Allislove** [comment]
I think, we need close this pull, i just created a new branch with the actual data in dev! and this works good ! without need of install react paginate!

**Pull request closed without merging**